package co.exception;

public class Aritmatic {
	public static void main(String[] args) {
		
		int a=10;
		int b =0;
		//int c = a/b;
		// System.out.println(c);//
		 try  
		 { int c=a/b;
		 }catch
			 (ArithmeticException e) {
				 System.out.println("bhiya yeh arithmatic exception h");
		
		 
		 
		 }
		System.out.println("hello himanshi");
	}
	

}
